src/components/Popups/Alerts/Alert.scss

```
@@ -87,8 +87,18 @@
     }
 
     .m-alert-confirm-button {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
       border-bottom-left-radius: var(--alert-border-radius);
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)